shared/chat/conversation/messages/loading-more.desktop.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from 'react'
+import {Box, Text} from '../../../common-adapters'
+import {globalStyles} from '../../../styles'
+
+import type {Props} from './loading-more'
+
+const MessageLoadingMore = ({style, loading}: Props) => (
```

Comment:
I think you should change the name of `loading` to `hasMoreItems`, I was a bit confused